docs/001_develop/03_client-capabilities/009_presentation/icon.mdx

```diff
@@ -14,21 +14,22 @@ import IconDemo from './examples/icon.js';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Icon
+This component enables you to show an icon.
 
 Use cases:
-* Displaying a specified icon.
-* An addition to presenting a listbox-option
+* displaying a specified icon
+* highlighting items in a listbox
 * cell-renderers and many others
 
 :::note
 You can use any free publicly-available icon from fontawesome. These are available via the links below:
 - [solid/regular](https://fontawesome.com/search?o=r&m=free) icons
 - [brand](https://fontawesome.com/search?o=r&m=free&f=brands) icons
-Paying for fontawesome pro will enable the use of all icons via this component also.
+Paying for fontawesome pro enables you to use all the fontawesome icons with this component.
```

Comment:
The formatting is broken here and this setence is now displayed on the same line as the bullet point above
![Screenshot 2025-01-16 at 10 27 50](https://github.com/user-attachments/assets/ff1135b2-a5e9-450b-bfeb-bf1cc513ac00)